terraform/modules/ecr/ecr-lifecycle-policy.json

```diff
@@ -0,0 +1,30 @@
+{
+    "rules": [
+      {
+        "action": {
+          "type": "expire"
+        },
+        "description": "Expire untagged images older than 14 days",
+        "rulePriority": 1,
+        "selection": {
+          "countNumber": 14,
+          "countType": "sinceImagePushed",
+          "countUnit": "days",
+          "tagStatus": "untagged"
+        }
+      },
+      {
+        "action": {
+          "type": "expire"
+        },
+        "description": "Keep only one tagged image, expire all others",
```

Comment:
Question...does this effectively purge an older image version as soon as a new one is downloaded? If so, should we consider bumping this count up to 2, in the event we accidentally push an update that affects a client's ability to work and we need to perform a rollback?